Mathlib/Topology/SubsetProperties.lean

```diff
@@ -880,10 +882,9 @@ theorem Inducing.isCompact_iff {f : α → β} (hf : Inducing f) {s : Set α} :
 /-- If `f : α → β` is an `Embedding` (or more generally, an `Inducing` map, see
 `Inducing.isCompact_iff`), the image `f '' s` of a set `s` is compact if and only if the set
 `s` is compact. -/
-theorem Embedding.isCompact_iff_isCompact_image {f : α → β} (hf : Embedding f) :
-    IsCompact s ↔ IsCompact (f '' s) :=
-  hf.toInducing.isCompact_iff.symm
-#align embedding.is_compact_iff_is_compact_image Embedding.isCompact_iff_isCompact_image
+theorem Embedding.isCompact_iff {f : α → β} (hf : Embedding f) :
+    IsCompact (f '' s) ↔ IsCompact s := hf.toInducing.isCompact_iff
```

Comment:
I should have mentioned that yesterday, but I think that we should actually keep the current order and reverse the order of `Inducing.isCompact_iff`. But if that's too annoying you can keep it like that and I'll fix it in a later PR (consistency between both versions is more important).

Reply:
Oh well... I have flipped this back, and also changed `Inducing.isCompact_iff`.
If you realize further inconsistencies, be invited to PR that yourself :-)